interop-testing/src/main/java/io/grpc/testing/integration/TestServiceServer.java

```diff
@@ -145,6 +153,18 @@ void start() throws Exception {
         .sslContext(sslContext)
         .addService(ServerInterceptors.intercept(
             TestServiceGrpc.bindService(new TestServiceImpl(executor)),
+            new ServerInterceptor() {
+              @Override
+              public <ReqT, RespT> Listener<ReqT> interceptCall(
+                  MethodDescriptor<ReqT, RespT> method,
+                  ServerCall<RespT> call,
+                  Metadata headers,
+                  ServerCallHandler<ReqT, RespT> next) {
+                call.setCompression("gzip");
```

Comment:
hardcoding the response won't honor the request, see https://github.com/grpc/grpc/blob/master/src/proto/grpc/testing/messages.proto#L92